src/hooks/index.ts

```diff
@@ -11,4 +11,6 @@ export type {UseOverlaySettings} from './useOverlay'
 export {useRenderForcingRef} from './useRenderForcingRef'
 export {useProvidedStateOrCreate} from './useProvidedStateOrCreate'
 export {useMenuInitialFocus} from './useMenuInitialFocus'
+export type {ComboboxCommitEvent} from './useCombobox'
+export {useCombobox} from './useCombobox'
```

Comment:
I didn't export `useSyntheticChange` because it feels more like an internal tool, but that may be debatable. It definitely could theoretically be useful outside of Primer.